src/pudl/metadata/resources/__init__.py

```diff
@@ -288,11 +288,32 @@
             "phone_number_2",
             "data_maturity",
         ],
+        "map_cols_dict": {
+            "owner_utility_id_eia": "utility_id_eia",
+            "owner_zip_code": "zip_code",
+            "owner_city": "city",
+            "owner_name": "utility_name_eia",
+            "owner_state": "state",
+            "owner_street_address": "street_address",
+        },
     },
 }
 """Columns kept for either entity or annual EIA tables in the harvesting process.
 
-For each entity type (key), the ID columns, static columns, and annual columns,
+For each entity type (key), the ID columns, static columns, annual columns, and mapped
+columns.
+
+Mapped columns allow for harvesting the same entity ID / value relationship
+from multiple columns in the same input dataframe. This is useful if an entity has a
+relationship with another entity of the same kind, for example owner and operator utilities
+showing up in the same ownership table records. The mapped column dictionary maps from
+column names of the second group of entity ID / value columns to the standard names
+for the entity ID / values columns.
+
+Mapped column dictionaries must include key value pairs for all the column names
+that should be harvested for that relationship, even if the column doesn't need to be renamed.
+If a column doesn't need to be renamed, it should map to itself,
+i.e. `utility_id_eia: utility_id_eia`.
```

Comment:
I added this functionality because the EIA ownership table has both owner and operator utilities in the same table, and we want to harvest both the owners and operators into the post-harvesting utilities table. This docstring could maybe be improved to be more clear.

Reply:
Did we also want to remove the `utility_id_eia` column from `ownership_eia860` and rename `owner_utility_id_eia` with `utility_id_eia` since it's poorly normalized as it is, and the operator ID can/should be merged in based on the operator associated with the plant ID instead?  This would probably require an update to the denormalized `ownership_eia860` asset.